src/config/test.yml

```diff
@@ -27,6 +27,7 @@ treeherder:
 
 try:
+  errorTaskUrl: "{{{host}}}{{{path}}}/raw-file/{{revision}}/error.yml"
```

Comment:
If I'm reading the above correctly, the error.yml files will be in the root directory of a repo. Should it live closer to other task YAML files? Or maybe in the corresponding product directories (browser, mobile, b2g, etc)?